lib/redux/constants/metrics.js

```diff
@@ -20,6 +20,23 @@ export const CLICK_GO_TO_BLIP = 'Clicked See Data in Blip';
 export const LOGIN_SUCCESS = 'Login Successful';
 export const LOGOUT_REQUEST = 'Clicked Log Out';
 
+export const CLINIC_LOGIN_SUCCESS = 'VCA Login Successful';
+export const CLINIC_SEARCH_DISPLAYED = 'VCA Search Screen Displayed';
+export const CLINIC_SEARCH_SELECTED = 'VCA Search Patient Selected';
+export const CLINIC_NEXT = 'VCA Search Next';
+export const CLINIC_ADD = 'VCA Search Add New';
+export const CLINIC_ADD_CANCEL = 'VCA Add Canceled';
+export const CLINIC_ADD_MRN = 'VCA Add MRN Saved';
+export const CLINIC_ADD_EMAIL = 'VCA Add Patient Email Saved';
+export const CLINIC_ADD_NEW_PATIENT = 'VCA Add New Patient Saved';
+export const CLINIC_ADD_INVALID_DATE = 'VCA Add Invalid Date';
+export const CLINIC_CHANGE_PERSON = 'VCA Change Person';
+export const CLINIC_EDIT_INFO = 'VCA Edit Info';
+export const CLINIC_DEVICE_STORED = 'VCA Device Stored';
+export const CLINIC_DEVICES_DONE = 'VCA Devices Done';
+export const CLINIC_TIMEZONE_SELECT = 'VCA Timezone Selected';
+export const CLINIC_CHANGE_DEVICES = 'VCA Change Devices';
```

Comment:
Some of these added metrics are not reflected in any tests and should be:
- [ ] CLINIC_SEARCH_SELECTED
- [ ] CLINIC_ADD_CANCEL
- [ ] CLINIC_CHANGE_PERSON
- [ ] CLINIC_EDIT_INFO
- [ ] CLINIC_DEVICES_DONE
- [ ] CLINIC_TIMEZONE_SELECT
- [ ] CLINIC_CHANGE_DEVICES

If there's some reason why (some or all of) these are particularly hard to instrument, let's discuss.

[Background: we've learned the hard way before that metrics are easy to lose if there aren't tests around validating their continued presence.]